scenes/game/game.gd

```diff
@@ -2405,6 +2411,8 @@ func can_press_ok():
 				return can_selected_cards_pay_force(select_card_require_force)
 			UISubState.UISubState_SelectCards_GaugeForEffect:
 				if select_card_must_be_max_or_min:
+					if instructions_cancel_allowed and len(selected_cards) == 0:
```

Comment:
I guess I don't get why this is needed.
How is this different from like Axl Bomber where it's a completely optional thing?

Reply:
It's something I tried adding just for the UX; without this you can press "OK" without selecting gauge and it does the same thing as pressing "Cancel" (now "Pass"), so I thought it would be clearer to have "OK" only be enabled when it would do something

Reply:
ok, that seems fine.  Just double check that like some other per_gauge effect and overall gauge effect works.
If anyone even has a per gauge effect...

Reply:
I think they're all on attacks atm (testament, chipp, axl), and I think I've checked all of them